core.md

```diff
@@ -0,0 +1,131 @@
+
+### Subprojects
+
+Subprojects are large tasks that the Project itself is undergoing.
+
+These may be:
+
+* Repositories under the Project's GitHub.
+* Large, multi-repository tasks that are being undertaken (ie. updating the API to a new version of the Game).
+* Smaller, more targetted tasks that are being undertaken (ie. refactoring the Modloader or API to remove or replace or rework a feature).
+
+### Teams
+
+The Project is split into four teams.
+
+A team is a group of people with similar goals, and equal weighting in most decision making - with some exceptions, which will be explored later.
+
+You may [apply to join some teams](https://links.neoforged.net/apply), but some require appointment from within an existing team, or further voting after application.
+
+#### Steering Council
+
+The Steering Council is the overall management and oversight of the Project. 
+
+They “steer” the project to keep it aligned with the long-term vision, hence its secondary name of Vision Team.
+
+The Council has at all times an odd number of active members, such that stalemates are not possible.
+
+A member may choose to become inactive and appoint another in their place until they return.
+
+At least one member of the Council is promoted from other teams via an annual internal vote, to ensure that the Council always has fresh insight from a new team member with a different view on situations, and thus does not stagnate over time.
+
+Once a year, [an internal voting process](https://peps.python.org/pep-8104/) will determine the members that will be promoted to the Steering Council, via the ranked voting system. It is intended that at least one, and at most three members will be promoted at a time.
+
+The Council also serves double duty as a mediator for disagreements between other teams. These shall be resolved by discussion, rather than by unanimous applications of the rules, so this is not in the purview of this document.
+
+However, the Council does not have all-encompassing powers. They are the avenue of last resort, and the vast majority of decisions should be made by the team in charge, not by the Steering Council.
+
+The Council has these powers to resolve the 1% of exceptional situations, not the 99%.
+
+#### Maintainers
+
+The Maintainers team consists of all project developers who have push access or technical rights (Write permissions to a Project's GitHub repository, or access to the Project's servers) to the project's repositories or infrastructure.
+
+They are the ones keeping the Project's repositories up to date, and ensuring the quality and standards of the Project are upheld, as well as adding new features as the community comes to require them.
+
+Though Maintainers may individually or collectively have different access rights across the project's many repositories or infrastructure services, all Maintainers within the team are considered equal to each other with respect to their rights within the team's governance.
```

Comment:
Seemingly contradictory language, assuming "their rights within the team's governance" refers to the definition above regarding it being those "who have push access or technical rights to the project's repositories or infrastructure."

Reply:
I don't see any contradiction here? You're going to have to be super extremely explicit and outline precisely what you mean.

Reply:
> Maintainers may individually or collectively have different access rights
and
> l Maintainers within the team are considered equal to each other with respect to their rights
within the same sentence.

Reply:
Access rights (permission to push to different repositories) and rights in the team (whose opinion is weighted more) are taken to be separate subjects entirely.

Reply:
You left out a key part of that second sentence (emphasis mine):

> _[...]_ all Maintainers within the team are considered equal to each other with respect to their rights **within the team's governance.**

That is, while Maintainers hold different levels of access to resources, repositories, or infrastruture, they have the same 'loudness of voice' and rights (e.g. voting rights) with each other in the Maintainers team.

Reply:
Shouldn't those who specialise in subprojects (such as subproject leaders) be given additional considerations based on their knowledge of a given area, rather than treating a new maintainer and an experienced specialist equally?

Reply:
Yes. This is why there's a separate topic for Subproject Leaders explicitly outlining their authority and rights exceeding standard maintainers.

Reply:
OK. this looks to be resolved to me. Paint wasn't understanding the context of access rights to repositories vs rights within the organization.